src/components/Watch.svelte

```diff
@@ -13,33 +13,34 @@
     chatZoom,
     showCaption,
     chatSplit,
+    displayMode
   } from '../js/store.js';
   import {
     paramsVideoId,
     VideoSide,
     ChatSplit,
     paramsContinuation,
     paramsIsVOD,
-    paramsEmbedded
+    DisplayMode
   } from '../js/constants.js';
   import ChatEmbed from './ChatEmbed.svelte';
   import Popout from './Popout.svelte';
   import Captions from './Captions.svelte';
 
   document.title = 'LiveTL';
-  let isResizing = false;
+  import { isResizing } from '../js/store.js';
```

Comment:
Put above the ``document.title`` line or use ``svelte:window`` to set the title.

Reply:
nvm you fixed this